.goreleaser.yml

```diff
@@ -1,25 +1,48 @@
-before:
-  hooks:
-    - make clean
-
 builds:
-  - binary: terraform-provider-snowflake_{{ .Tag }}
-    env:
-      - CGO_ENABLED=0
-    goos:
-      - darwin
-      - linux
-    goarch:
-      - amd64
-    ldflags:
-      - "-w -s -X github.com/chanzuckerberg/terraform-provider-snowflake/pkg/version.GitSha={{.Commit}} -X github.com/chanzuckerberg/terraform-provider-snowflake/pkg/version.Version={{.Version}} -X github.com/chanzuckerberg/terraform-provider-snowflake/pkg/version.Dirty={{.Env.DIRTY}}"
+- env:
+  - CGO_ENABLED=0
+  goos:
+    - freebsd
+    - openbsd
+    - solaris
+    - windows
+    - linux
+    - darwin
+  goarch:
+    - amd64
+    - '386'
+    - arm
+    - arm64
+  ldflags:
+    - "-w -s -X github.com/chanzuckerberg/terraform-provider-snowflake/pkg/version.GitSha={{.Commit}} -X github.com/chanzuckerberg/terraform-provider-snowflake/pkg/version.Version={{.Version}} -X github.com/chanzuckerberg/terraform-provider-snowflake/pkg/version.Dirty={{.Env.DIRTY}}"
+  ignore:
+    - goos: darwin
+      goarch: '386'
+    - goos: openbsd
+      goarch: arm
+    - goos: openbsd
+      goarch: arm64
+  binary: '{{ .ProjectName }}_v{{ .Version }}'
 
 archives:
-  - files:
-    - none*
+  - format: zip
+    name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}'
 
-release:
-  prerelease: false
+checksum:
+  name_template: '{{ .ProjectName }}_{{ .Version }}_SHA256SUMS'
+  algorithm: sha256
 
-env_files:
-  github_token: ~/.config/goreleaser/github_token
```

Comment:
are these needed?

Reply:
I have no idea of their purpose. Looks like you added it [here](https://github.com/chanzuckerberg/terraform-provider-snowflake/pull/103)

Reply:
I have alwayst just had a token exported in my environment though.